dspy/retrieve/chromadb_rm.py

```diff
@@ -21,12 +25,19 @@
 except ImportError:
     chromadb = None
 
+try:
+    import torch
```

Comment:
please don't import dependencies that are not in requirements.txt in the top level

you can import them inside the method you need, so people who don't need them don't see errors for them